qiita_db/user.py

```diff
@@ -280,16 +277,36 @@ def verify_code(cls, email, code, code_type):
             return False
 
         db_code = db_code[0]
+        if not db_code:
+            raise QiitaDBError("No %s code for user %s" %
```

Comment:
Can you add this exception to the `Raises` section of the docstring? Also ... just noticed that `code_type` does not have a description, would you mind adding that as well?